src/Service.Tests/dab-config.MySql.json

```diff
@@ -1,1307 +1,2972 @@
 {
-  "$schema": "../../schemas/dab.draft.schema.json",
+  "$schema": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json",
   "data-source": {
     "database-type": "mysql",
-    "connection-string": "server=localhost;database=dab;Allow User Variables=true;uid=root;pwd=REPLACEME"
```

Comment:
why remove `Allow User Variables=true`?

Reply:
https://github.com/Azure/data-api-builder/commit/495175b5e7f05c834a281bcc7225ae6c80769e35 is where it changed in the config generator, we just haven't updated this.